misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/CharSequenceAssertionsSpec.kt

```diff
@@ -212,4 +214,22 @@ abstract class CharSequenceAssertionsSpec(
             }
         }
     }
+
+    describeFun(mismatches.name){
+        val mismatchesFun = mismatches.lambda
+
+        context("text '$text"){
+            // TODO : Matches Negative Condition
```

Comment:
is there a reason why you keep the TODO? (same question applies to the TODO below)

Reply:
I kept it for better understanding of the flow. I was not supposed push that. I thought I removed all the TODO statements. But I forgot. Fixed now